docs/backend-system/core-services/auditor.md

```diff
@@ -78,10 +76,77 @@ In this example, an audit event is created for each request to `/my-endpoint`. T
+- **`meta.queryType` (or related field) for Specific Actions within a Group:** Use a `meta` field (like `queryType`, `actionType` or similar) to specify the particular action or query that occurred within the broader `eventId` group.
+  - For instance, with `eventId: entity-fetch`, use `meta: { queryType: 'by-id' }` to represent fetching an entity by its ID. Other examples could be:
+    - `meta: { queryType: 'all' }` for fetching all entities.
+    - `meta: { queryType: 'by-query' }` for fetching entities by a query.
+    - `meta: { actionType: 'delete' }` for `eventId: entity-mutate` when an entity was deleted.
+    - `meta: { actionType: 'create' }` for `eventId: location-mutate` when a location was added.
+  - Use `meta` fields to add more context to the event being tracked.
+- **Avoid Redundant Prefixes:** Do not include redundant prefixes related to the plugin ID in your event names. The plugin context is already provided separately.
+- **Clear and Concise:** Choose names that clearly and concisely describe the event being audited.
+
+## Common Meta Keys and Values
+
+The following table details common keys found within the `meta` object of audit events and their formats:
+
+| Key           | Description                                                                    | Format                                | Example(s)                                                                |
+| ------------- | ------------------------------------------------------------------------------ | ------------------------------------- | ------------------------------------------------------------------------- |
+| `queryType`   | Specifies the type of query performed when fetching data.                      | A kebab-case string                   | `all`, `by-id`, `by-name`, `by-query`, `by-refs`, `ancestry`, `by-entity` |
+| `actionType`  | Specifies the type of action performed when modifying data.                    | A kebab-case string                   | `create`, `delete`, `refresh`                                             |
+| `entityRef`   | The full reference of an entity, including kind, namespace, and name.          | `[kind]:[namespace]/[name]`           | `component:default/my-component`, `group:my-org/team-a`                   |
+| `locationRef` | A specific reference to a location being operated on.                          | Any string representing the location. | `Url:https://example.com/catalog-info.yaml`, `custom:default/my-location` |
+| `uid`         | The unique identifier of a location or other object involved in the operation. | Any valid unique ID string            | `9a4e740b-e557-427f-b9f2-0d4f092b1c1e`                                    |
+
+By following these conventions, you create a more structured and informative audit trail that is easier to search, filter, and understand. This allows you to better group and understand the events being logged.
```

Comment:
Just wondering, will this render okay in the docs site? Not sure I've ever seen / tried it 🤔

Reply:
Yup, this is fine @schultzp2020 👍